src/Http/ServerRequest.php

```diff
@@ -1112,10 +1113,11 @@ public function accepts(?string $type = null)
      * of the accepted content types.
      *
      * @return array An array of `prefValue => [content/types]`
+     * @deprecated 4.4.0 Use accepts() or ContentTypeNegotiation instead.
```

Comment:
```suggestion
     * @deprecated 4.4.0 Use `accepts()` or `ContentTypeNegotiation` class instead.
```

Reply:
Should probably use the qualified name instead of just name + class.